CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 This page documents production updates to Cloud Code for Visual Studio Code. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## Version 1.4.0 Release Notes
+
+### New features
+
+* **Kubernetes Context Management:** Contexts are now accessible from the Kubernetes Explorer!
```

Comment:
You can embed the screen capture images in this markdown similarly to how it was done below.